server/php-master.cpp

```diff
@@ -363,6 +365,11 @@ struct pipe_info_t {
   connection pending_stat_queue;
 };
 
+enum class WorkerType {
```

Comment:
On one hand `worker_info_t` must contain field identifying type of worker which is one out of two: `http_worker` or `task_worker`, it can't be `master`. On the other hand `generic_event_loop` can be called by master process as well, so it should accept such option. So `ProcessType` wouldn't be enough.